bench/macro/lsm-tree-bench-wp8.hs

```diff
@@ -394,13 +396,14 @@ doRun gopts opts = do
               | otherwise      = sequentialIterations
         time <- timed_ $
           benchmarkIterations
-            check
+            (\b y -> check b y >> when (b `mod` (opts.batchCount `div` 100) == 0) (putChar '.'))
```

Comment:
Pull the progress calc out so we don't repeat it every time.

I'm not sure this is the best place to put it. I've also got a tidy-up branch where I separate the `check` into a `shouldOutput :: Bool` and `doOutput` action, to really minimise the effect when we're not doing testing. But progress reporting should be unconditional.

Reply:
I'm not sure I understand what you would like this to look like. Wouldn't we need to compute the `mod` to know if we've made progress? Or should we print a dot by default?

Reply:
The `mod` is fine. Just pull out and name the `progress = opts.batchCount `div` 80`. It's clearer that way.